### ISSUE TITLE
Pirate Bay Streaming

### ISSUE THREAD

**Issue by losdelrock** [jmarth/plugin.video.kmediatorrent#80]
Hello, thanks for the great plug in!   It all works well with one issue.  On the Pirate Bay, it starts downloading then stays on "Working" for about a minute.  It then tries to play but returns to the menu.  If I set it to download before playing then the movie downloads and it plays fine.  I have also set the buffer size higher and no change.  All the other sites work which would point to a PB issue.  Any ideas?

**Comment by jmarth** [comment]
Strange. Seems to be working fine for me. But it might be a torrent2http issue and I am working on replacing it with something else

**Comment by losdelrock** [comment]
Thanks for the quick reply....
I am using a Minix Android box, is there any way I can help by sending logs etc?
Also what version of XBMC?  I think Minix has it's own version and it may be a beta.
Adrian.




Sent from my Samsung device

-------- Original message --------

**Comment by jmarth** [comment]
Yes please upload a log. It could be internet speed, no room for the file or many other things.

**Comment by losdelrock** [comment]
Plenty of room for the files, 1TB USB drive with 800GB free.  Internet Speed 50Mb and it downloads the streams very quickly - just when it comes to opening them perhaps the 1st few seconds play then it quits playing and reverts to the torrent list page.  

There is always a very long wait on the "Working" spinner, sometimes even a minute.  Sometimes not even the 1st few seconds play,  All the other sources work.  I used to have XBMC Torrent installed and I used XBMC Backup to restore the settings - could something have been left behind in the settings?

Also, where are the logs so I can upload them?

Thanks for your help.

Adrian.

**Comment by jmarth** [comment]
You can use this link to get the file http://kodi.wiki/view/Log_file

**Comment by losdelrock** [comment]
Thank you, doing some digging, if it as AVI file, the movie starts but then after a couple of seconds it stops.  If it as an mp4 it dos not even start but I get a message "One or more files failed to open, please check log file"

Which log file would the information be in, I can have a quick look and see if any of the error messages mean anything?

Adrian.

**Comment by jmarth** [comment]
The avi issue us known and I'm working in it. You can find his to get the log file from the link above

**Comment by losdelrock** [comment]
OK, I upgraded to Kodi 13.3.3 and now I am able to get the message on mp4 torrents from TPB.  I get "File Cannot Fast Start" so then the file downloads and plays fine.  The issue still exists with AVI files.  Just FYI I am running the add on on a Minix Neo X7 running Andoird 4.4.2.  I will report back any other findings and log files.
